tests/test_coded_location.py

```diff
@@ -81,6 +81,43 @@ def test_coded_location_helper():
     assert coded_loc_lores.lon == 171.1, "Should have rounded longitude"
 
 
+@pytest.mark.parametrize(
```

Comment:
ref https://docs.oracle.com/cd/E49933_01/server.770/es_dev/src/cdg_sorting_geosort.html

 - latitude is the latitude of the location in whole and fractional degrees. Valid values are from a minimum of -90.0 to a maximum of 90.0 (inclusive). Positive values indicate north latitudes and negative values indicate south latitudes.
 - longitude is the longitude of the location in whole and fractional degrees. Valid values are from a minimum of -180.0 to a maximum of 180.0 (inclusive). Positive values indicate east longitudes and negative values indicate west longitudes.